droidlet/tools/artifact_scripts/compute_checksum.py

```diff
@@ -56,9 +58,12 @@ def compute_checksum_for_directory(agent=None, artifact_type=None, model_name=No
         checksum_name = "datasets.txt"
 
     artifact_path = os.path.join(ROOTDIR, "droidlet/artifacts", artifact_folder_name)
-    checksum_write_path = os.path.join(
-        ROOTDIR, "droidlet/tools/artifact_scripts/tracked_checksums/" + checksum_name
-    )
+    if checksum_write_path is None:
```

Comment:
I think this commit is out-of-date right now. After yesterday's meeting, I converted this function back. Right now there is no changes. 

Previously, I added ```checksum_write_path``` since I imported it for ```compute_checksum_tar_upload``` which stores hash in checksum.txt. The default ```compute_checksum``` method only saves hash to files under ```tracked_checksums```. ```checksum_write_path``` specifies where to the computed hash.